app/models/user.rb

```diff
@@ -24,4 +29,18 @@ def unfollow(followed_id)
   def following?(followed_id)
     active_relationships.find_by(followed_id: followed_id).present?
   end
+
+  # ポストをお気に入りするメソッド
+  def favorite(post_id)
+    favorites.create(post_id: post_id)
+  end
+  # ポストのお気に入りを解除するメソッド
+  def unfavorite(post_id)
+    pp post_id
+    favorites.find_by(post_id: post_id).destroy!
+  end
+  # お気に入りしているかどうかを確認するメソッド
+  def favoriting?(post_id)
```

Comment:
exists? や nil? のようにするといいです。
```suggestion
  def favorite?(post_id)
```